ramls/records-editor.raml

```diff
@@ -0,0 +1,72 @@
+#%RAML 1.0
+
+title: QuickMARC Editor Records
+version: v1.0
+protocols: [ HTTP, HTTPS ]
+baseUri: http://github.com/folio-org/mod-quck-marc
+
+documentation:
+  - title: QuickMARC record API
+    content: This documents the API calls that can be made to work with MARC records
+
+types:
+  marcUi: !include schemas/marc_ui.json
+  errors: !include raml-storage/raml-util/schemas/errors.schema
+  UUID:
+    type: string
+    pattern: ^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[1-5][0-9a-fA-F]{3}-[89abAB][0-9a-fA-F]{3}-[0-9a-fA-F]{12}$
+
+traits:
+  validate: !include raml-storage/raml-util/traits/validation.raml
+
+/records-editor/records:
+  get:
+    queryParameters:
```

Comment:
I wasn't able to generate correct RecordsEditorRecords interface (target/generated-sources/raml-jaxrs/org/folio/rest/jaxrs/resource/RecordsEditorRecords.java) with such declaration. Could you please introduce 2 separate query parameters and just make them non-required
```
    queryParameters:
      id:
        description: UUID of the MARC record
        type:        UUID
        required:    false
        example:     f6c574b5-b1d5-4e5e-a28b-161b7e03e81a
      per_page:
        description: UUID of the instance that is related to the MARC record
        type:        UUID
        required:   false
        example:     f6c574b5-b1d5-4e5e-a28b-161b7e03e81a     
```
As a result the following signature should be generated:
```
void getRecordsEditorRecords(@QueryParam("id") @Pattern(regexp = "^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[1-5][0-9a-fA-F]{3}-[89abAB][0-9a-fA-F]{3}-[0-9a-fA-F]{12}$") String id, @QueryParam("instanceId") @Pattern(regexp = "^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[1-5][0-9a-fA-F]{3}-[89abAB][0-9a-fA-F]{3}-[0-9a-fA-F]{12}$") String instanceId, Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext);
```

Reply:
Done